tests/devices/test_default_qubit_jax.py

```diff
@@ -438,7 +438,7 @@ def circuit(x):
                 qml.RZ(x, wires=w, id="x")
             return qml.expval(qml.PauliZ(wires=0))
 
-        with pytest.raises(ValueError, match="Sum of amplitudes-squared does not equal one."):
+        with pytest.raises(ValueError, match="The state must be a vector of norm 1.0"):
```

Comment:
Why this is changed?

Reply:
While checking the same thing in both templates, each of those returned a different message. When unifying them I had to choose one. As you can see the meaning is equivalent